Resources/Prototypes/Entities/Clothing/Head/helmets.yml

```diff
@@ -246,4 +246,55 @@
         Blunt: 0.5
         Slash: 0.5
         Piercing: 0.5
-        Heat: 0.9
\ No newline at end of file
+        Heat: 0.9
+  - type: IdentityBlocker
+
+- type: entity
+  parent: ClothingHeadEVAHelmetBase
+  id: ClothingHeadHelmetVoidParamed
+  name: Paramedic Void Helmet
+  description: A void helmet made for paramedics.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/paramedhelm.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/paramedhelm.rsi
+  - type: IngestionBlocker
+  - type: TemperatureProtection
+    coefficient: 0.1
+  - type: Armor
+    modifiers:
+      coefficients:
+        Slash: 0.95
+        Heat: 0.90
+        Radiation: 0.75
+
+- type: entity
+  parent: ClothingHeadEVAHelmetBase
+  id: ClothingHeadHelmetAncient
+  name: NTSRA void helmet
+  description: An ancient space helmet, designed by the NTSRA branch of CentCom.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/ancientvoidsuit.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/ancientvoidsuit.rsi
+  - type: IngestionBlocker
+  - type: Tag
+    tags:
+    - HelmetEVA
+
```

Comment:
Revert

Reply:
I may have made a mistake in reverting this, I think I committed some stuff which shouldn't have been?